qa/sys/debian/bootstrap.sh

```diff
@@ -5,3 +5,4 @@ apt-get update
 apt-get install -y openjdk-8-jdk
 update-alternatives --config java
 update-alternatives --config javac
+sudo update-ca-certificates -f
```

Comment:
for the way we run this scripts it will run for all debians, we might also not need this for all debian's, would be nice to check if this happen for others, and if not, basically only do this for ubuntu-1404.